Bottom: 0680c87779f79bea7dbdd4026df387c65da7160e
Top:    7aa2ca5c13569ebea8f10b03f9c6926b12b17c82
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-11-06 17:19:27 +0000

IMPORTANT: DONT DRAW POUR OUTLINES DURING EXPORT

They can produce shorted boards otherwise!


---

diff --git a/src/draw.c b/src/draw.c
index 49f011f..1c1a504 100644
--- a/src/draw.c
+++ b/src/draw.c
@@ -708,7 +708,8 @@ pour_callback (const BoxType * b, void *cl)
   struct pin_info *i = (struct pin_info *) cl;
   PourType *pour = (PourType *)b;
 
-  DrawPour (i->Layer, pour, 0);
+  if (gui->gui)
+    DrawPour (i->Layer, pour, 0);
 
   if (pour->PolygonN)
     {
